plugins/modules/systemd_creds_encrypt.py

```diff
@@ -32,38 +32,43 @@
   name:
     description:
       - The credential name to embed in the encrypted credential data.
+        When using the O(dest) option, you can set this to an empty
+        string ("") to prevent C(systemd-creds encrypt) from using
+        filename from O(dest) as the credential name. Conversely,
+        leave out the O(name) option to let C(systemd-creds encrypt)
+        use the filename from O(dest) as the credential name.
```

Comment:
I think this note might be better placed in `dest` below - both cases described required that `dest` is set.

Also, maybe a more affirmative wording might help. Something like (inserted be low in dest's description):
```yaml
- When this is set and O(name=""), then the credential_name is XXXXX (not clear to me the right answer here)
- When this is set and O(name) is not passed, then credential name will take the value of O(dest).
- When this is set and O(name) is also set with a non-empty name, then credential name will take the value of O(name) (???? will it???)
```
Or similar - take this with a pinch of salt.

Reply:
What do you think?